osu.Game/Graphics/UserInterface/OsuButton.cs

```diff
@@ -21,9 +22,9 @@ namespace osu.Game.Graphics.UserInterface
     /// </summary>
     public class OsuButton : Button
     {
-        public string Text
+        public LocalisableString Text
         {
-            get => SpriteText?.Text;
+            get => SpriteText.Text;
```

Comment:
Intentionally removed?

Reply:
Because `LocalisableString` is a readonly struct, adding the null permissiveness back as it was would make this a `LocalisableString?` which is hugely breaking (code lens shows 108 usages). Maybe we want

```csharp
public LocalisableString Text
{
	get => SpriteText?.Text ?? default;
	// ...
}
```

as the original PR did.